config/settings.yml

```diff
@@ -13,6 +13,8 @@ globus:
   uploads_directory: /uploads/
   transfer_endpoint_id: endpoint_uuid
   help_doc_url: https://docs.google.com/document/d/10b7y3yZCOfyVJ_uP4l7QHbkILQKnfdJi9hj-MpmmdIk
+  test_mode: false # for testing purposes in non-production only, simulates globus API calls
+  test_user_exists: true # if test_mode=true, simulates if the globus user exists
```

Comment:
Should the default value be `false`?

Reply:
the default value for `test_mode` absolutely should be false, the second value is going to toggle back and forth depending on the workflow you want to test (and is irrelevant unless `test_mode = true` anyway)

Reply:
OK, sounds good. :)